### DISCUSSION TITLE
Demo for Armada

### DISCUSSION THREAD

**Discussion by kannon92** [armadaproject/armada#2447] [General]
I created a simple demo for Kubecon.  

Demo walks through some `armadactl`, lookout and airflow.

https://github.com/armadaproject/armada/assets/3780425/06f10057-1531-44cc-9abc-a8fde3817a39

**Comment by mladjan-gadzic** [comment]
hey @kannon92! thanks for this demo, i found it useful as a new user.

at 0:50 you are submitting a job to armada. is there a way to pass job id or it is only created by armada? for my usecase i'd need to know job id before submitting if possible.

**Reply by kannon92** [reply]
cc @dejanzele. I am afraid I don't remember this answer anymore.
